### ISSUE TITLE
Add Acs Alias Script errors d'jour

### ISSUE THREAD

**Issue by jborgione** [agrc/acs-alias-address-point#5]
I have tried to add the arcpy.GetParameterAsText() lines in and it errors out saying "variable arcpy.GetParameterAsText" not defined.

So I hard code the paths and it errors out saying that my output directory does not exist.

Sigh.

**Comment by jborgione** [comment]
I figured out the GetParameters issue; user error on the syntax.  But  creating an output directory is not working:  If I give it an existing directory, it errors out with:

<class 'arcgisscripting.ExecuteError'>: Failed to execute. Parameters are not valid.
ERROR 000732: File GDB Location: Dataset U:\Weber911\test\output does not exist or is not supported
Failed to execute (CreateFileGDB).

If I give it directory that does not exist it errors out the same way:

<class 'arcgisscripting.ExecuteError'>: Failed to execute. Parameters are not valid.
ERROR 000732: File GDB Location: Dataset U:\Weber911\test\output1 does not exist or is not supported
Failed to execute (CreateFileGDB).

Here is  what I added after commenting out the roads  =  ps.path.join(blah blah blah) lines:

roads = arcpy.GetParameterAsText(1)
    addressPoints = arcpy.GetParameterAsText(0)
    outputDirectory = arcpy.GetParameterAsText(2)

Here is a screen cap of the parameters.  Notice the output path ends at 'test'.  It used to be U:Weber911\test\output until the script clobbered the existing direcort "output".  Notice also the warning...
![parmeters](https://cloud.githubusercontent.com/assets/10454364/5725873/0473ff98-9b14-11e4-9e85-d29439f1b207.PNG)
![warning](https://cloud.githubusercontent.com/assets/10454364/5725874/074a3f0c-9b14-11e4-9926-3efaf30f0d4e.PNG)
